packages/@aws-cdk/assertions/lib/private/outputs.ts

```diff
@@ -1,7 +1,7 @@
 import { StackInspector } from '../vendored/assert';
 import { filterLogicalId, formatFailure, matchSection } from './section';
 
-export function findOutputs(inspector: StackInspector, logicalId: string, props: any = {}): { [key: string]: any }[] {
+export function findOutputs(inspector: StackInspector, logicalId: string, props: any = {}): { [key: string]: any } {
```

Comment:
Wondering if this should be `{ [key: string]: { [key: string]: any } }`? (in all places in this PR)

WDYT?

Reply:
Good point. This will allow for checks like `expect(assert.Foo).toEqual({})`.